whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -48,15 +48,59 @@ <h4 id='error_tag' class='light warning'></h4>
         {{ vendor_form.story }}
         <p class=" field_text">In Port:</p>
         {{ vendor_form.status }}
-        <h3>Location</h3>
-        <p class="field_text">Street Address*:</p>
+        <h3>Location</h3><br>
+
+    <!-- Code for Map based location inputs start -->
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
+
+    <script type="text/javascript" src='http://maps.google.com/maps/api/js?sensor=false&libraries=places'></script>
+    <script src="http://rawgit.com/Logicify/jquery-locationpicker-plugin/master/dist/locationpicker.jquery.js"></script>
+
+<body>
+   <div class="form-horizontal">
+        <div class="form-group">
+            <div><input type="text" class="form-control" id="us2-address"/></div>
+        </div><br>
+
+        <div id="us2" style="width: 550px; height: 400px;"></div><br>
```

Comment:
Styles should be defined in the style.scss file, which should then be compiled into a style.css file. (When I'm feeling lazy I just add it to both and call it good.)